GoogleForCreators/web-stories-wp
# Dashboard: Add global Page Advancement setting

## Context
Adds Page Advancement global option to the Dashboard settings which will be used if the setting hasn't been set per story.


## Summary



## Relevant Technical Choices



## To-do


- [x] What to do with the migration V11?



## User-facing changes
<img width="824" alt="Screenshot 2022-10-13 at 12 58 37" src="https://user-images.githubusercontent.com/3294597/195566662-71907726-e762-42e7-8ed1-222a4dc73c8d.png">



## Testing Instructions






This PR can be tested by following these steps:

1. Go to Dashboard settings
2. Change the Page Advancement settings
3. Verify that when adding a new story, the settings configured in the Dashboard are now used
4. Verify existing stories will not be influenced.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



Two new tracking values: `change_auto_advance` and `change_default_page_duration`


### Does this PR have a legal-related impact?



## Checklist




---



Fixes #11528
